services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -96,8 +96,14 @@ public void deleteProject(MigrationProject migrationProject)
     }
 
     @Override
+    public void deleteOldProvisionalProjects()
+    {
+        this.migrationProjectService.deleteOldProvisionalProjects(0);
```

Comment:
As explained at the meeting, this is a bit more complicated that in seems at first glance.
0 minutes is there because otherwise you would get a duplicate name project warning.
I've then taken the other measure to fix - ignoring the provisional flag by `getProjectByName()`.